jerry-core/ecma/builtin-objects/typedarray/ecma-builtin-float32array-prototype.inc.h

```diff
@@ -31,6 +31,10 @@ NUMBER_VALUE (LIT_MAGIC_STRING_BYTES_PER_ELEMENT_U,
               4,
               ECMA_PROPERTY_FIXED)
 
+/* Routine properties:
+*  (property name, C routine name, arguments number or NON_FIXED, value of the routine's length property) */
+ROUTINE (LIT_MAGIC_STRING_TO_STRING_UL, ecma_builtin_float32array_prototype_object_to_string, 0, 0)
+ROUTINE (LIT_MAGIC_STRING_JOIN,  ecma_builtin_float32array_prototype_join, 1, 1)
```

Comment:
These ROUTINES should not be defined per typedarray version, but simply in the typedarray-prototype file. Please see the `ecma_builtin_typedarray_prototype_every` implementation on how this can be done. By putting the implementation in the correct place we can reduce the code duplications.